collectors/feature/gitlab/src/main/java/com/capitalone/dashboard/collector/FeatureSettings.java

```diff
@@ -0,0 +1,59 @@
+package com.capitalone.dashboard.collector;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+/**
+ * Bean to hold settings specific to the Feature collector.
+ * 
+ */
+@Component
+@ConfigurationProperties(prefix = "gitlab")
```

Comment:
It might, however the settings should be the same for both collectors.  So I don't see an issue unless you are using a different server for your gitlab SCM and gitlab issues.  But I could easily alter the name to avoid the conflict.